src/city/constants.h

```diff
@@ -15,7 +15,8 @@ typedef enum {
     ADVISOR_ENTERTAINMENT = 9,
     ADVISOR_RELIGION = 10,
     ADVISOR_FINANCIAL = 11,
-    ADVISOR_CHIEF = 12
+    ADVISOR_CHIEF = 12,
+    ADVISOR_HOUSING = 19
```

Comment:
The intention was that sub-advisors could be linked to parent advisor via the use of modulo 13, as in the code I used to ensure the pop advisor button is highlighted when the housing inspector is invoked:

```
    for (int i = 0; i < 13; i++) {
        int selected_offset = 0;
        if (current_advisor && i == current_advisor - 1) {
        if (current_advisor && i == (current_advisor % 13) - 1) {
            selected_offset = 13;
        }
        image_draw(image_group(GROUP_ADVISOR_ICONS) + i + selected_offset, 48 * i + 12, 441);
```

This way, this would all work if a later change was made to add a new screen to the Trade advisor, for example, by assigning the new advisor to ([the integer representing the trade advisor] + 13).

Reply:
13 being the magic number as there are 12 advisors + 1 null in the zero index position.